### ISSUE TITLE
kms.teevee.asia: suspicious domain

### ISSUE THREAD

**Issue by MEtICaRT** [hagezi/dns-blocklists#3484]
### Which domain(s) should be blocked?

kms.teevee.asia

### Why should these domain(s) be blocked?

My Windows PC repeatedly attempts to connect to this link, but it is blocked by my TLD (`.asia` blocked) settings. I suspect this behavior started after activating Windows using a third-party activation script. Could someone please check what this link is? If it’s suspicious or dangerous, could you block it?

### Privacy

- [X] I confirm that the report does not contain any private information.

**Comment by hagezi** [comment]
```
Domain:
 - kms.teevee.asia OK

Malware/Phishing/Scam:
 - Threat?
   HaGeZi.TIF      NO
   HaGeZi.TIF.RAW  NO
   ThreatFox       NO
   URLhaus         NO
   ThreatView      NO
   USOM            NO
   CERT.PL         NO

 - Scam/Phishing?
   Phishing.Army   NO
   PT/OP/PH        NO
   Jarelllama SCAM NO

 - NRD?
   NRD 30          NO

Top 1M:
 - Umbrella:       YES
 - Cloudflare:     NO
 - Tranco:         NO
 - Majestic:       NO
 - BuiltWith:      NO
 - Chrome:         NO

Top 10M:
 - DOMCOP:         NO

Secure DNS:
 - 360Secure       OK
 - AliDNS          OK
 - CFIEC           OK
 - CleanBrowsing   OK
 - Cloudflare      OK
 - ComodoSecure    OK
 - CONTROLD.TIF    OK
 - DNS0.eu         OK
 - DNS0.eu.ZERO    OK
 - DNSWatchGO      OK
 - HaGeZi.TIF      OK
 - Neustar         OK
 - NextDNS.TIF_AI  OK
 - NortonCS        OK
 - NRD.DGA.IDN     OK
 - Quad9           OK
 - SafeDNS         OK
 - UltraDNS        OK
 - Umbrella        OK
 - YandexSafe      OK

Blocklists:
 - 1Hosts.Lite     OK
 - 1Hosts.Mini     OK
 - 1Hosts.Pro      OK
 - AdGuardDNS      OK
 - CONTROLD.AT     OK
 - DevDansHosts    OK
 - EasyList        OK
 - GoodbyeAds      OK
 - HaGeZi.LIGHT    OK
 - HaGeZi.NORMAL   OK
 - HaGeZi.PRO      OK
 - HaGeZi.PRO.PLUS OK
 - HaGeZi.TIF      OK
 - HaGeZi.ULTIMATE OK
 - hBlock          OK
 - OISD.Big        OK
 - OISD.Small      OK
 - QuidsUp.NOTRACK OK
 - StevenBlack     OK

Intels:
 - Google          https://transparencyreport.google.com/safe-browsing/search?url=kms.teevee.asia
 - VirusTotal      https://www.virustotal.com/en/domain/kms.teevee.asia/information/
 - AlienVault      https://otx.alienvault.com/indicator/domain/kms.teevee.asia
 - Bitdefender     https://trafficlight.bitdefender.com/info/?url=https%3A%2F%2Fkms.teevee.asia
 - FortiGuard      https://www.fortiguard.com/webfilter?q=kms.teevee.asia&type=&engine=1
 - Kaspersky       https://opentip.kaspersky.com/kms.teevee.asia?tab=web
 - McAfee          https://siteadvisor.com/sitereport.html?url=kms.teevee.asia
 - Norton          https://safeweb.norton.com/report/show?url=kms.teevee.asia
 - OpenDNS         https://domain.opendns.com/kms.teevee.asia
 - URLVoid         https://www.urlvoid.com/scan/kms.teevee.asia/
 - Yandex          https://yandex.com/safety/?l10n=en&url=kms.teevee.asia
 - ThreatMiner     https://www.threatminer.org/domain.php?q=kms.teevee.asia

```

**Comment by hagezi** [comment]
These “KMS” tootls should all be treated with caution.

**Comment by hagezi** [comment]
Since I don't know why the domain is being contacted and what is being transferred, I can't say anything about it. Unfortunately, you can't tell from the domain alone. When it is called up, it is redirected to a blocked domain page.

I have blocked other kms domains and will block these as well.

**Comment by iam-py-test** [comment]
Based on some quick searching, it seems this may have been a legitimate Key Management Services (KMS) server<sup><a href="https://archive.is/2NVcg">[1]</a><a href="https://www.tenforums.com/general-support/179248-fix-windows-10-all-cummulative-updates-not-installing.html">[2]</a></sup> (along with hosting some tutorials - [example](https://web.archive.org/web/20211105115739/https://kb.teevee.asia/tips-and-tricks/how-to-fix-office-365-shows-as-office-2019-professional-plus-office-365-showed-as-office-2019/)), but currently it's parked.
You can run `reg query "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion\SoftwareProtectionPlatform" /V KeyManagementServiceName` to see what your system's Key Management Service is. You can use `slmgr /skms [whatever domain]` to change it to another server.
Thanks
